components.py

```diff
@@ -1,11 +1,9 @@
 import dash
```

Comment:
I suggest we call it controls instead of components. ui_components was also better than components. Components could be anything at all.